app/views/wishlist/index.html.haml

```diff
@@ -1 +1,19 @@
 %h1 Wishlist
+= form_tag wishlist_path, role: :search, id: :wishlist_course_form do
+  = hidden_field_tag :course_id
+  = text_field_tag :course_title, nil, placeholder: "Quickly add courses to your wishlist", class: 'form-control'
```

Comment:
I usually prefer conciseness for placeholders. Perhaps "Add course to wishlist"?